CONTRIBUTING.md

```diff
@@ -79,4 +79,4 @@ The pipeline requires sign offs immediately prior to the deployments to Test-PyP
 
 The `DEV_VERSION` variable is to work around the PyPI behaviour where uploads are immutable and published immediately. This means that each upload to PyPI 'uses up' that particular version (on that particular PyPI instance). Since we wish to deploy exactly the same bits to both Test-PyPI and PyPI, without this workaround the version released to PyPI would depend on the number of issues discovered when uploading to Test-PyPI. If PyPI updates their release process to separate the 'upload' and 'publish' steps (i.e. until a package is published, it remains hidden and mutable) then the code associated with `DEV_VERSION` should be removed.
 
-As part of the release process, the `build-wheels.ps1` script uses `process_readme.py` to turn all the relative links in the ReadMe file into absolute ones (this is the reason why the applied tag has be of the form `v[_base_version]`). The `process_readme.py` script is slightly fragile with respect to the contents of the ReadMe, so after significant changes its output should be verified.
\ No newline at end of file
+As part of the release process, the `build_wheels.py` script uses `process_readme.py` to turn all the relative links in the ReadMe file into absolute ones (this is the reason why the applied tag has be of the form `v[_base_version]`). The `process_readme.py` script is slightly fragile with respect to the contents of the ReadMe, so after significant changes its output should be verified.
```

Comment:
Mainly because the links in the checked-in ReadMe are all relative. So when it goes to PyPI, we change all those to full URLs pointing at GitHub. The script should also make sure that the 'latest version' URL is properly set (that's the bit most likely to be fragile).

Reply:
@adrinjalali how do other projects handle this?

Reply:
I don't see why those URLs shouldn't be absolute in the first place, see https://github.com/scikit-learn/scikit-learn/blob/master/README.rst as an example maybe.

Reply:
That's because they should reference the files of that version of the package. If CONTRIBUTING.md undergoes major changes from v0.5 to v0.6 then we wouldn't want the old version to point to the new markdown file. Even worse, some files may disappear/get renamed, which would mean the link gets lost.

Reply:
In reality the main files shouldn't really disappear, though. Curious what @riedgar-ms  and @MiroDudik  think, but I think it doesn't need to be addressed in this PR.

Reply:
started a thread in Teams, resolving this one

Reply:
I see. I still find it very odd. But we can talk about the build process maybe another time :D I really believe it can be simplified. One thing I noticed checking this, is that the pypi README links to the one on github, instead of somewhere in the docs.

Ideally the readthedocs documentation should include these, and the links should point to that one, not github.